<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>571</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="actionVerLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout" name="firstCoderGrid">
       <item row="0" column="0">
        <widget class="QLabel" name="descFirstCoderLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;First coder setting&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolBox" name="toolBox">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <widget class="QWidget" name="hemming">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>567</width>
            <height>92</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Hemming</string>
          </attribute>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>0</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Size of packege</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="sizePackHemEdit">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>0</y>
             <width>111</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>120</number>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="cyclical">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>567</width>
            <height>92</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Cyclical</string>
          </attribute>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>0</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Size of packege</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Cyclic Polynomial</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="sizePackCycEdit">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>0</y>
             <width>111</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>120</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="poliCycEdit">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>30</y>
             <width>111</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>13</number>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="convolution">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>567</width>
            <height>92</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Convolutional</string>
          </attribute>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>0</y>
             <width>101</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>List of polynomials</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="listConvEdit">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>0</y>
             <width>161</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>3, 5, 7</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>131</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Count register of memory</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="countRegConvEdit">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>30</y>
             <width>161</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="fountain">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>567</width>
            <height>82</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Fountain</string>
          </attribute>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>0</y>
             <width>81</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Size of package</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>25</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Size of block</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>51</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Count of blocks</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="sizePackFounEdit">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>1</y>
             <width>121</width>
             <height>20</height>
            </rect>
           </property>
           <property name="maximum">
            <number>99999999</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
           <property name="displayIntegerBase">
            <number>10</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="sizeBlockFounEdit">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>26</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
           <property name="maximum">
            <number>99999999</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="countBlockFounEdit">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>51</y>
             <width>121</width>
             <height>20</height>
            </rect>
           </property>
           <property name="maximum">
            <number>99999999</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
          <zorder>label_7</zorder>
          <zorder>label_8</zorder>
          <zorder>label_9</zorder>
          <zorder>toolBox_3</zorder>
          <zorder>sizePackFounEdit</zorder>
          <zorder>sizeBlockFounEdit</zorder>
          <zorder>countBlockFounEdit</zorder>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="secondCoderGrid">
       <item row="0" column="0">
        <widget class="QLabel" name="descSecondCoderLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Second coder setting&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolBox" name="toolBox_2">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="hemming_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>567</width>
            <height>92</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Hemming</string>
          </attribute>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>0</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Size of packege</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="sizePackHemEdit_2">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>0</y>
             <width>111</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>120</number>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="cyclical_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>567</width>
            <height>83</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Cyclical</string>
          </attribute>
          <widget class="QLabel" name="label_12">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>0</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Size of packege</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_13">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Cyclic Polynomial</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="sizePackCycEdit_2">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>0</y>
             <width>111</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>120</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="poliCycEdit_2">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>30</y>
             <width>111</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>13</number>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="convolution_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>567</width>
            <height>92</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Convolutional</string>
          </attribute>
          <widget class="QLabel" name="label_14">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>0</y>
             <width>101</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>List of polynomials</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="listConvEdit_2">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>0</y>
             <width>161</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>3, 5, 7</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_15">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>131</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Count register of memory</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="countRegConvEdit_2">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>30</y>
             <width>161</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="fountain_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>567</width>
            <height>92</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Fountain</string>
          </attribute>
          <widget class="QLabel" name="label_16">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>0</y>
             <width>81</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Size of package</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_17">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>25</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Size of block</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_18">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>51</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Count of blocks</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="sizePackFounEdit_2">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>1</y>
             <width>121</width>
             <height>20</height>
            </rect>
           </property>
           <property name="maximum">
            <number>99999999</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
           <property name="displayIntegerBase">
            <number>10</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="sizeBlockFounEdit_2">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>26</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
           <property name="maximum">
            <number>99999999</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="countBlockFounEdit_2">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>51</y>
             <width>121</width>
             <height>20</height>
            </rect>
           </property>
           <property name="maximum">
            <number>99999999</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="testCoderGrid">
       <item row="1" column="2" colspan="2">
        <widget class="QSpinBox" name="countTestEdit">
         <property name="maximum">
          <number>99999999</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Count test:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="beginNoiseLable">
         <property name="text">
          <string>Noise start:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="noiseFinishEdit">
         <property name="locale">
          <locale language="C" country="AnyCountry"/>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>50.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="noiseStartEdit">
         <property name="locale">
          <locale language="C" country="AnyCountry"/>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>50.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="endNoiseLable">
         <property name="text">
          <string>Noise end:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Information for test:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QLineEdit" name="informationEdit">
         <property name="text">
          <string>985</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Тестирование первого кодера&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Тестирование каскада кодеров&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
